src/components/main/flight/CheckoutWithFlightScreen.js

```diff
@@ -1,94 +1,99 @@
-import React from 'react'
+import React from "react";
```

Comment:
Also not checking this one. 
It'd be preferable if you could also remove this from the commit since I see there ARE some changes, but it's best to have them done in a new file after this one is deprecated.

Reply:
The changes to previous files have to do with passing the distance prop when it was not passed previously.

Reply:
Okay, that makes sense. Checkout WILL be merged with carbonemissions screen later on though so since receipt is getting the distance, I'm not going review this screen yet.